evm/src/trace/node.rs

```diff
@@ -91,6 +94,34 @@ impl CallTraceNode {
         }
     }
 
+    pub fn geth_trace(&self, opts: GethDebugTracingOptions) -> GethTrace {
+        GethTrace {
```

Comment:
This is great, perhaps longer term there could be a `GethTracer` trait so that the `callTracer` could also be implemented and the tracer type could be pulled from the `opts`